.env.example

```
@@ -107,10 +109,31 @@ REACT_APP_TCAD_MATIC_BRIDGE_CONTRACT_ADDRESS=0x0
 REACT_APP_TGBP_MAINNET_BRIDGE_CONTRACT_ADDRESS=0x0
 REACT_APP_TGBP_MATIC_BRIDGE_CONTRACT_ADDRESS=0x0
 
+# --------------------------------
+# --- bridge fees and minimum ----
+# --------------------------------
+REACT_APP_MATIC_FLOOR_FLAT_FEE=1
+REACT_APP_MATIC_CEILING_FLAT_FEE=2
+
+REACT_APP_BRIDGE_MINIMUM_AMOUNT_USD=20
+
+
+# -----------------------------------
+# --- bridge addresses testing ------
+# -----------------------------------
+REACT_APP_MOCK_TOKEN_MAINNET=0x0
+REACT_APP_MOCK_TOKEN_MATIC=0x0
+REACT_APP_MOCK_MAINNET_BRIDGE_CONTRACT_ADDRESS=0x0
+REACT_APP_MOCK_MATIC_BRIDGE_CONTRACT_ADDRESS=0x0
 
 # ------------
 # --- misc ---
 # ------------
 
+REACT_APP_SHOW_DEMO=
+
 REACT_APP_IS_BRIDGE_CAPPED=false
+
+REACT_APP_IS_CONSOLE_LOG_ENABLED=
```

Should we add a default?

```suggestion
REACT_APP_IS_CONSOLE_LOG_ENABLED=true
```